lib/cheetah.rb

```diff
@@ -691,6 +701,11 @@ def build_result(streams, status, options)
       res
     end
 
+    def allowed_exitstatus?(options)
+      # more exit status allowed for non array  or non empty array
```

Comment:
Just an extra space:
```suggestion
      # more exit status allowed for non array or non empty array
```